dockerfiles/dockerfile.obr

```diff
@@ -0,0 +1,7 @@
+ARG dockerRepository
+ARG tag
+
+FROM ${dockerRepository}/galasadev/galasa-managers:${tag}

```

Comment:
Please can you update this to `FROM ${dockerRepository}/galasa-dev/managers-maven-artefacts:${tag}` as this is where the new Managers image in GHCR is.

Reply:
done